utils.go

```diff
@@ -39,6 +39,12 @@ func valueOrReadFrom(value string, file *os.File) string {
 	if err != nil {
 		log.Fatal("Error while reading from file", file, err)
 	}
+
+	// remove newline added to std in from command execution
```

Comment:
how can you be sure that the newline is not part of the secret?

Reply:
It seems to be consistent for reading from standard in, as the return from executing the command is included after the file. I'm not sure as to the why, can't find much about it. Ofcourse if we read from an actual file (say with a -file option or something) we don't want this to happen.